pkgs/development/python-modules/ignite/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, mock
+, pytorch
+, pynvml
+, scikitlearn
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "ignite";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "pytorch";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "15k6dd11yxn4923llcpmw4srl1by5ljhh7aw5pnkn4n4qpywh6cm";
+  };
+
+  checkInputs = [ pytest mock ];
+
+  checkPhase = "pytest -k 'not visdom and not tensorboard and not mlflow and not polyaxon' tests/";
```

kind of hard to parse when all crammed in one line. not really sure if this is much better, but it is a little
```suggestion
  checkPhase = ''
    pytest test/ -k 'not visdom and not tensorboard and not mlflow and not polyaxon'
  '';
```

---

Makes sense.